config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
 
+  get 'items/create'
+
+  get 'items/destroy'
```

Comment:
This one also is not typical. Did you mean to use a `resources :items` call instead?